internal/voting/sql.go

```diff
@@ -4,192 +4,198 @@ package voting
-	FROM ` + votingSchemaName + `.resolutions AS r
-	INNER JOIN ` + votingSchemaName + `.resolution_types AS t ON r.type = t.id
-	LEFT JOIN ` + votingSchemaName + `.votes AS v ON r.id = v.resolution_id
-	LEFT JOIN ` + votingSchemaName + `.voters AS vr ON v.voter_id = vr.id
+		r.vote_body_proposer AS vote_body_proposer
+	FROM ` + VotingSchemaName + `.resolutions AS r
+	INNER JOIN ` + VotingSchemaName + `.resolution_types AS t ON r.type = t.id
+	LEFT JOIN ` + VotingSchemaName + `.votes AS v ON r.id = v.resolution_id
+	LEFT JOIN ` + VotingSchemaName + `.voters AS vr ON v.voter_id = vr.id
 	WHERE r.expiration <= $1
-	GROUP BY r.id, r.body, t.name, r.expiration, r.vote_body_proposer, r.extra_vote_id
+	GROUP BY r.id, r.body, t.name, r.expiration, r.vote_body_proposer
 	ORDER BY r.id;` // order by id for determinism. ids are unique in the result.
 
 	// getFullResolutionInfo gets the full info of a resolution.
 	// it redundantly returns the id for convenience with functions that consume this query
 	getFullResolutionInfo = `
 	SELECT r.id AS id, r.body AS body, t.name AS type, r.expiration AS expiration,
 		SUM(vr.power) AS approved_power, ARRAY_AGG(int8send(vr.power) || vr.name ORDER BY vr.id) AS voters,
-		r.vote_body_proposer AS vote_body_proposer, r.extra_vote_id AS extra_vote_id
-	FROM ` + votingSchemaName + `.resolutions AS r
-	INNER JOIN ` + votingSchemaName + `.resolution_types AS t ON r.type = t.id
-	LEFT JOIN ` + votingSchemaName + `.votes AS v ON r.id = v.resolution_id
-	LEFT JOIN ` + votingSchemaName + `.voters AS vr ON v.voter_id = vr.id
+		r.vote_body_proposer AS vote_body_proposer
+	FROM ` + VotingSchemaName + `.resolutions AS r
+	INNER JOIN ` + VotingSchemaName + `.resolution_types AS t ON r.type = t.id
+	LEFT JOIN ` + VotingSchemaName + `.votes AS v ON r.id = v.resolution_id
+	LEFT JOIN ` + VotingSchemaName + `.voters AS vr ON v.voter_id = vr.id
 	WHERE r.id = $1
-	GROUP BY r.id, r.body, t.name, r.expiration, r.vote_body_proposer, r.extra_vote_id
+	GROUP BY r.id, r.body, t.name, r.expiration, r.vote_body_proposer
 	ORDER BY r.id;` // order by not necessary since only one result?
 
-	allVoters                      = `SELECT name, power FROM ` + votingSchemaName + `.voters;`
-	getResolutionByTypeAndProposer = `SELECT r.id FROM ` + votingSchemaName + `.resolutions AS r
-	INNER JOIN ` + votingSchemaName + `.resolution_types AS t ON r.type = t.id
+	allVoters                      = `SELECT name, power FROM ` + VotingSchemaName + `.voters;`
+	getResolutionByTypeAndProposer = `SELECT r.id FROM ` + VotingSchemaName + `.resolutions AS r
+	INNER JOIN ` + VotingSchemaName + `.resolution_types AS t ON r.type = t.id
 	WHERE t.name = $1 AND vote_body_proposer = $2
 	ORDER BY r.id;` // order by id for determinism
 )
+
+// upgrades V0 -> V1
+const (
+	dropHeightTable = `DROP TABLE IF EXISTS ` + VotingSchemaName + `.height`
+)
+
+// upgrades V1 -> V2
+const (
+	dropExtraVoteID = `ALTER TABLE ` + VotingSchemaName + `.resolutions DROP COLUMN extra_vote_id;`
```

Comment:
We could combine these, but I only see downsides, no upsides.  Fine like this IMO

Reply:
yeah, lets keep them this way.